src/explore-education-statistics-common/src/modules/table-tool/components/TableSizeError.tsx

```diff
@@ -0,0 +1,77 @@
+import Button from '@common/components/Button';
+import ErrorPrefixPageTitle from '@common/components//ErrorPrefixPageTitle';
+import downloadService from '@common/services/downloadService';
+import { Subject } from '@common/services/tableBuilderService';
+import React, { useEffect, useRef } from 'react';
+
+interface Props {
+  focus: boolean;
+  releaseId?: string;
+  showDownloadOption?: boolean;
+  subject: Subject;
+}
+
+const TableSizeError = ({
+  focus = false,
+  releaseId,
+  showDownloadOption = true,
+  subject,
+}: Props) => {
+  const handleDownloadFile = async (fileId: string) => {
+    if (releaseId) {
+      await downloadService.downloadFiles(releaseId, [fileId]);
+    }
+  };
```

Comment:
This is pretty simple, so I'd suggest we just inline this into the `onClick` prop itself.

Reply:
Happy that this has been addressed :)